Formula/pyside.rb

```diff
@@ -19,41 +19,24 @@ class Pyside < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "llvm" => :build
+  depends_on "ninja" => :build
+  depends_on "llvm"
   depends_on "python@3.9"
   depends_on "qt"
 
   def install
-    ENV.remove "HOMEBREW_LIBRARY_PATHS", Formula["llvm"].opt_lib
```

Comment:
Let's keep this, CMake might be better behaved about not linking with LLVM using this.

Reply:
I just gave that a try. It doesn't remove the dependency, which isn't surprising since the shiboken tool depends on libclang. I think the only way to remove the LLVM runtime depependency  would be to put the shiboken tool in a separate package. I'm not entirely sure how to separate the runtime from the tool in a clean way in that case.

Reply:
Does it not work with the apple provided llvm?

Reply:
No, they don't provide the headers and libraries, just the compiler.